cpp/src/ast/expressions.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2020-2021, NVIDIA CORPORATION.
```

Comment:
Not sure if this file is new or renamed. If it is new, please update header year.

Reply:
Good catch. Updated.